packages/web-app-files/src/mixins/actions/fetch.js

```diff
@@ -23,8 +28,14 @@ export default {
     }
   },
   methods: {
-    $_fetch_trigger(filePath, mimetype) {
-      const url = this.$client.helpers._webdavUrl + filePath
+    $_fetch_trigger(file, mimetype, isPublicFile) {
+      if (isPublicFile) {
+        const url = this.$client.helpers.instance + file.downloadURL.substring(1)
```

Comment:
Please use the server url from the store.
Add `import { mapGetters } from 'vuex'` as import and `...mapGetters(['configuration']),` within the computed properties. The you can use the server url like this: `this.configuration.server`. It is guaranteed to have a trailing slash.
The `$client` naturally has the same url, because it is initialized with it. But you should grab it from the source, not from an intermediate object.

Reply:
And yes, it is already super ugly to use the webdav url like this from the client. ;-) But that can be fixed later. For the server url it's a quick win.